AssetManagementTestProject/TestCases/APITest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using AssetManagementTestProject.DAO;
+using AssetManagementTestProject.Services;
+using AssetManagementTestProject.TestSetup;
+using NUnit.Framework;
+using ServiceStack;
+
+namespace AssetManagementTestProject.APITest
+{
+    [TestFixture]
+    public class APITest : NUnitWebTestSetup
+    {
+        public CreateUserService? CreateUserService;
+        [Test]
+        public void AdminCanCreateUser()
+        {
+            CreateUserService = new CreateUserService();
+            CreateUserDAO.CreateUserResponse expectedNewUsers = CreateUserService.GetNewUsers(Token);
+        }
```

Comment:
Remove this test after you are done testing, as this test is not related to other test cases